README.md

```diff
@@ -1,2 +1,462 @@
+
+
+### certificate_requests
+
+**Note:** Fields such as `common_name`, `country`, `state`, `locality`,
+`organization`, `organizational_unit`, `email`, `key_usage`, and
+`extended_key_usage` that can be included in the certificate request
+are defined by the RFC 5280.
+
+**Note:** Be aware that the CA might not honor all the requested fields.
+For example, even if a request include `country: US`, the CA might issue
+the certificate without `country` in it's subject.
+
+**Note:** The fields `dns`, `email` and `ip` are used to define the Subject
+Alternative Names (SAN).
+
+
+| Parameter            | Description                                                                                       | Type        | Required | Default                 |
+|----------------------|---------------------------------------------------------------------------------------------------|:-----------:|:--------:|-------------------------|
+| name                 | Name of the certificate. A full path can be used to choose the directory where files will be stored.| str       | yes      | -                       |
+| ca                   | CA that will issue the certificate. See [CAs and Providers](#cas-and-providers).                  | str         | yes      | -                       |
+| dns                  | Domain (or list of domains) to be included in the certificate. Also can provide the default value for [common\_name](#common_name). | str or list | no | - |
+| email                | Email (or list of emails) to be included in the certificate.                                      | str or list | no       | -                       |
+| ip                   | IP, or list of IPs, to be included in the certificate. IPs can be IPv4, IPv6 or both. Also can provide the default value for [common\_name](#common_name). | str or list | no | - |
+| auto_renew           | Indicates if the certificate should be renewed automatically before it expires.                   | bool        | no       | yes                     |
+| owner                | User name (or user id) for the certificate and key files.                                         | str         | no       | *User running Ansible*  |
+| group                | Group name (or group id) for the certificate and key files.                                       | str         | no       | *Group running Ansible* |
+| key\_size            | Generate keys with a specific keysize in bits.                                                    | int         | no       | 2048 - See [key\_size](#key_size) |
+| common\_name         | Common Name requested for the certificate subject.                                                | str         | no       | See [common\_name](#common_name)  |
+| country              | Country requested for the certificate subject.                                                    | str         | no       | -                       |
```

Comment:
Though a minor thing...,  safer to be "Country code" ?

Reply:
done